raiden/transfer/architecture.py

```diff
@@ -119,6 +120,11 @@ class Event:
     pass
 
 
+class EventsError(NamedTuple):
+    events: List[Event]
+    error: str
```

Comment:
How do I interpret an instance of this?  Does one of the `events` contain a problem, or all `events` together form a problem?

Reply:
Are `events` to be applied to the state machine?  If yes, after already queued events, or after already stacked events?